app/migrations/20170320154519_missions.js

```diff
@@ -6,6 +6,7 @@ exports.up = knex =>
     table.integer('casefile_id').references('casefiles.id').onDelete('CASCADE');
     table.integer('user_id').references('users.id').onDelete('CASCADE');
     table.string('url');
+    table.boolean('last_id');

```

Comment:
I haven't yet fixed the thing with the ids not always being what I expect them. I had been getting the last id by getting all the rows in the missions table and getting the length, but that doesn't work if the ids are not in order or some have been deleted. So as a workaround I added a boolean value, last_id, to the table to mark which one is last. When a new one is added the current last one will go to false and the new one true. Though I don't think I added that yet. *scurries off to fix it*